Add `ECSTask` block to infrastructure documentation (#6806)
PrefectHQ/prefect: docs/concepts/infrastructure.md

| stream_output | Bool indicating whether to stream output from the subprocess to local standard output. |

> ### ECSTask

Reviewer: ```suggestion
### ECSTask <span class="badge experimental"></span>
```

> [`ECSTask`](https://prefecthq.github.io/prefect-aws/ecs/) infrastructure runs your flow in an ECS Task.

Reviewer: ```suggestion

[`ECSTask`](https://prefecthq.github.io/prefect-aws/ecs/) infrastructure runs your flow in an ECS Task.
```

> > ⚠️ Note this infrastructure block is experimental. The intefaces within may change without notice.

Reviewer: ```suggestion
!!! note "Experimental block"
    This infrastructure block is experimental. The interfaces of this block may change without notice.
```

Requirements for `ECSTask`:

> - The ephemeral Orion API won't work with ECS directly. You must have an Orion or Prefect Cloud API endpoint set in your [agent's configuration](/concepts/work-queues/).

Reviewer: ```suggestion
- The ephemeral Prefect Orion API won't work with ECS directly. You must have a Prefect Orion or Prefect Cloud API endpoint set in your [agent's configuration](/concepts/work-queues/).
```

> - `prefect-aws` [collection](https://github.com/PrefectHQ/prefect-aws) must be installed within the agent environment: `pip install prefect-aws`

Reviewer: ```suggestion
- The `prefect-aws` [collection](https://github.com/PrefectHQ/prefect-aws) must be installed within the agent environment: `pip install prefect-aws`
```

> - the `ECSTask` and `AwsCredentials` blocks must be registered within the agent environment: `prefect block register -m prefect_aws.ecs`

Reviewer: ```suggestion
- The `ECSTask` and `AwsCredentials` blocks must be registered within the agent environment: `prefect block register -m prefect_aws.ecs`
```
